pkg/api/k8s_versions.go

```diff
@@ -128,7 +128,7 @@ var k8sComponentVersions = map[string]map[string]string{
 		"aci-connector":                    "virtual-kubelet:latest",
 		ContainerMonitoringAddonName:       "oms:ciprod01092019",
 		AzureCNINetworkMonitoringAddonName: "networkmonitor:v0.0.5",
-		"cluster-autoscaler":               "cluster-autoscaler:v1.3.4",
+		"cluster-autoscaler":               "cluster-autoscaler:v1.3.7",
```

Comment:
Based on your findings let's definitely keep this change.

Reply:
why not add 1.3.7 to the VHD script (even if we don't build it right away)?

Reply:
Because of cranial flatulence? I've added it to the VHD script now. 😄